src/roiextractors/extractors/miniscopeimagingextractor/__init__.py

```diff
@@ -11,4 +11,4 @@
     An ImagingExtractor for the Miniscope video (.avi) format.
 """
 
-from .miniscopeimagingextractor import MiniscopeImagingExtractor
+from .miniscopeimagingextractor import MiniscopeImagingExtractor, MiniscopeMultiSegmentExtractor
```

Comment:
Is this a typo? This is still an imaging extractor and not a segmentation right?

Reply:
It is a type. Let me fix it, thanks for all the other fixes as well!

Reply:
@weiglszonja thanks for pointing this out.  Now that I think about it, MultiSegment is also a bad name because in Spikeinterface multi-segment recordings do not extract the different recordings within the session as a single continuous recording. I settled for `MiniscopeMultiRecordingImagingExtractor` to avoid this possible confusion.

Can you take a look at the new naming and the new docstrings to see if it makes sense?